global.json

```diff
@@ -1,5 +0,0 @@
-{
-  "sdk": {
-    "version": "2.1.0-preview1-007002"
```

Comment:
maybe we should instead lock to 2.0.0 (the release version)? Do you know why this was originally added?

Reply:
I have no idea, I guess it was before 2.0.0 was released. yes we should probably  lock and downgrade the fake script.

This was just to get rid of the very annoying blocker locally.

Reply:
I have no idea, I guess it was before 2.0.0 was released. yes we should probably  lock and downgrade the fake script.

This was just to get rid of the very annoying blocker locally.